vmdb/app/views/layouts/listnav/_container_project.html.haml

```diff
@@ -19,5 +19,47 @@
           %li
             = link_to("#{ui_lookup(:table => "ems_container")}: #{@record.ext_management_system.name}",
                 {:controller => "ems_container", :action => 'show', :id => @record.ext_management_system.id.to_s},
-                {:title => _("Show this container service's parent %s") % ui_lookup(:table => "ems_container")},
-                '/images/icons/16/link_external.gif')
\ No newline at end of file
+                {:title => _("Show this container service's parent %s") % ui_lookup(:table => "ems_container")})
+          - if role_allows(:feature => "container_group_show_list")
+            - num_groups = @record.number_of(:container_groups)
+            - if num_groups == 0
```

Comment:
@simon3z all of these are cut and paste as well.
Could you introduce a helper for these so there isn't 4 copies of this same code?